### ISSUE TITLE
[Card] styles bugs

### ISSUE THREAD

**Issue by oliviaflory** [carbon-design-system/carbon-for-ibm-dotcom#5552]
## Detailed description

> Describe in detail the issue you're having.

## Styling bugs within Card
Wrapping all styling bugs into one ticket for Cards, but happy to break out into separate issues if that is better for dev.

### 1. Focus
Focus state is thicker than 2px and has rounded corners in [Web components](https://ibmdotcom-web-components.mybluemix.net/?path=/story/components-card--default) storybook, should be 2px per the specs and $focus token and square corner.

- [x] 2px thick focus
- [x] no rounded corner
- [x] card with image focus should be around _entire_ card  
- [ ] inverse card focus should have white border to be

| Expected | Web components |
| --- | --- |
| ![Screen Shot 2021-03-19 at 4.24.20 PM.png](https://images.zenhubusercontent.com/5cacfd2d41f23a1e95e068dc/b23d6f2c-689b-476b-9e24-9bdeceb5fdae) | ![Screen Shot 2021-03-19 at 4.24.10 PM.png](https://images.zenhubusercontent.com/5cacfd2d41f23a1e95e068dc/32a68e6f-2df7-4e43-b319-b1bae71f4d7d) |
|![Screen Shot 2021-03-19 at 4.29.10 PM.png](https://images.zenhubusercontent.com/5cacfd2d41f23a1e95e068dc/057e92e3-730e-41e4-8d27-27a5dc07feeb) | ![Screen Shot 2021-03-19 at 4.24.44 PM.png](https://images.zenhubusercontent.com/5cacfd2d41f23a1e95e068dc/b87ebf32-077e-4280-b60a-428058b3b398) |

### 2. Hover
Images in cards are not getting hover state

- [ ] default image darkens 8% on hover
- [ ] inverse image lightens 8% on hover

![Screen Shot 2021-03-19 at 4.32.24 PM.png](https://images.zenhubusercontent.com/5cacfd2d41f23a1e95e068dc/cf191aa7-2739-4e37-b493-c253360c6b22)

### 3. Inverse CTA
- [x] CTA & icon should be $inverse-link token (CTA is using inverse token in card group, so may just need to add styling within inverse card by default?)
![Screen Shot 2021-03-19 at 4.35.18 PM.png](https://images.zenhubusercontent.com/5cacfd2d41f23a1e95e068dc/3847792b-5c99-42f4-b33b-e2a46e2c1d8b)

### 4. Eye brow text – Gray 100 theme
Eyebrow text is rendering smaller on gray 100 theme, possibly not getting the expressive theme? This came up in a QA call, but have not seen another issue to address

-  [x] Fix eyebrow text size in gray 100 theme

| Gray 90 | Gray 100 |
| --- | --- |
| ![Screen Shot 2021-03-19 at 5.02.09 PM.png](https://images.zenhubusercontent.com/5cacfd2d41f23a1e95e068dc/ecd4d996-9a8b-4da8-9442-2d986e01b118) | ![Screen Shot 2021-03-19 at 5.02.25 PM.png](https://images.zenhubusercontent.com/5cacfd2d41f23a1e95e068dc/0c0f8360-a9f6-4091-9d49-8e59425dccd0) |

### 5. Body text token
- [x] body-long-02

### 6. CTA hover
- [x] no hover on CTA text, seeing in web components 

![Mar-19-2021 17-21-27.gif](https://images.zenhubusercontent.com/5cacfd2d41f23a1e95e068dc/257e60a5-1880-41de-848a-037e0c06ffbf)
